code/__HELPERS/game.dm

```diff
@@ -53,6 +53,24 @@
 	for(var/I in adjacent_turfs)
 		. |= get_area(I)
 
+/**
+ * Get a bounding box from a list of turfs.
+ *
+ * Returns: list(x1, y1, x2, y2)
+ */
+/proc/get_bbox_of_turfs(list/items)

```

Comment:
```suggestion
/proc/get_bbox_of_turfs(list/atoms_in_range)
```
Something like this would be more clear. `item` is a common type, and could be misleading. Same for the other var.